### PR TITLE
Enable K3s Install for KRIB

### PR DESCRIPTION
**Pull request by zehicle**
This also makes the containerd task more generic
Also removes the hideme meta data to prepare for actual secure params

Some other minor cleanups

### CODE REVIEW

**Comment by zehicle** [comment]
video https://www.youtube.com/watch?v=6bgBn6Ze6jg !

**Comment by zehicle** [comment]
address @galthaus concerns: removed krib/k3s param
also removed the password mess.

**Comment by zehicle** [comment]
retested with updates.  Looks good for k3s and k8s

File: krib/templates/containerd-install.sh.tmpl [hunk 1]
```diff
@@ -2,11 +2,15 @@
...
 set -e
 
+echo "starting containerd v{{.Param "containerd/version"}} install"
 # Get access and who we are.
 {{template "setup.tmpl" .}}
 [[ $RS_UUID ]] && export RS_UUID="{{.Machine.UUID}}"
 
+# not sure why we need this here...but it needs to be protected
+{{ if .ParamExists "etcd/controller-ip" }}
 ETCD_CONTROLLER_IP={{.Param "etcd/controller-ip"}}
```
Comment (galthaus) [inline krib/templates/containerd-install.sh.tmpl]: This is not needed - just delete it

**Pull request merged**